testing/tests/DevExpress.ui/collectionWidgetParts/editingTests.js

```diff
@@ -1469,26 +1469,34 @@ module('deleting of items', () => {
         });
     });
 
-    test('deleteItem should trigger delete callback only once with correct itemData', function(assert) {
+    test('onDeleteItem should trigger delete callback only once with correct itemData', function(assert) {
         const item = '0';
-        let deleteActionFlag = 0;
-        let deletedItem = null;
-
+        const deleteActionSpy = sinon.spy();
         const $element = $('#cmp');
         const instance = new TestComponent($element, {
             items: [item],
-            onItemDeleted: function(e) {
-                deleteActionFlag++;
-                deletedItem = e.itemData;
-                assert.equal(e.itemIndex, 0, 'correct index specified');
-            }
+            onItemDeleted: deleteActionSpy
+        });
+
+        instance.deleteItem(instance.itemElements().eq(0));
+
+        assert.strictEqual(deleteActionSpy.callCount, 1, 'itemDeleted triggered 1 time');
+        assert.strictEqual(item, deleteActionSpy.firstCall.args[0].itemData, 'item equals selected item');
+    });
+
+    test('deleteItem event should trigger delete callback only once with correct itemData', function(assert) {
+        const item = '0';
+        const deleteActionSpy = sinon.spy();
+        const $element = $('#cmp');
+        const instance = new TestComponent($element, {
+            items: [item]
         });
```

Comment:
`$element` is used only once and can be inlined.
```suggestion
        const instance = new TestComponent('#cmp', {
            items: [item]
        });
```